src/components/KonnectorFolder.jsx

```diff
@@ -1,25 +1,87 @@
-import React from 'react'
+import React, { Component, PureComponent } from 'react'
+import classNames from 'classnames'
+
+import Icon from 'cozy-ui/react/Icon'
 import { translate } from 'cozy-ui/react/I18n'
 
 import styles from '../styles/konnectorFolder'
 
 import DescriptionContent from './DescriptionContent'
 
-class KonnectorFolder extends React.Component {
-  render({ t, driveUrl, trigger }) {
+const ERRORED = 'ERRORED'
+const FETCHING = 'FETCHING'
+const SUCCESS = 'SUCCESS'
+
+/**
+ * Cache
+ */
+let driveUrl
+
+/**
+ * Renders a link only if href prop is provided
+ * @extends PureComponent
+ */
+class SmartLink extends PureComponent {
```

Comment:
Nit: SmartLink seems a bit too generic, I think [MaybeLink](http://hackage.haskell.org/package/base-4.12.0.0/docs/Data-Maybe.html) would be better, Maybe is the word used in languages supporting Optionals (value that can be there or undefined).

Reply:
Thanks I spent some time looking for an explicit name 👍